documentation/processing.tex

```diff
@@ -26,16 +22,60 @@ \subsection{Overview}
 	\item \emph{Processing tracer} for tracing processed statements (see \cref{macro_tracer}).
 \end{itemize}
 
-\subsectionSrc{Types of statement processors}{parser\_library/src/processing/statement\_processors/statement\_processor.h}
-\label{lab06:sect_proc}
 
+\subsection{Composition}
+
+As the processing of the HLASM source file is rather complicated, we define a complex set of abstraction objects over the Gordian assembling of HLASM language:
+
+\begin{description}
+	\item[Statement provider] (see \cref{lab06:sect_prov}) is able to produce \TT{statement} structures. Its functionality is to provide statements from its finite statement stream.
```

Comment:
je niekde definovane co je finite statement stream? "provide statements from statement stream" neznie very helpful

Reply:
nie je to definovane. chcem povedat ze je to nejaka zasebou iduca postupnost statementov (stream) a ze je konecna (finite).
podla mna ta veta je helpful, citatel sa ma o co zrazu opriet a predstavit si to